adding section on plagiarising content (#2017)
mdn/content: README.md

You can use <https://whatsdeployed.io/s/DLi/mdn/content> to see if your change
has been deployed to the production site.

### Can I copy content from other sources to put on MDN

In general, we do not approve of copying content from other sources and putting
 it on MDN. MDN should be made up of original content wherever possible. If we

>   receive a pull request and discover that it contains [plagiarised]() content,

Reviewer: should there be a link here?

Author: I was going to link to a definition of plagiarism, but I decided not to in the end, so I've removed the link syntax in my next update.

   we will close it and request that the submitter resubmit the change with
    the content rewritten into their own words.

> If there is a genuine justification for taking content from other sources and
>  putting it on MDN (for example if someone wants to donate an article to MDN
>   that they previously published on their blog, or if it makes sense to copy a
>    complex reference sheet to MDN that is already available somewhere else), 
>    then you should first discuss this with the MDN team:

Reviewer: ```suggestion
If someone wants to donate an article to MDN that they previously published on their blog or it makes sense to copy a complex reference sheet to MDN there may be justification for republishing it. In these cases you should discuss your plan with the MDN team first:
```

Reviewer: just tried to break up that massive sentence a bit.

Reviewer 2: Drive-by comment: It looks like there is an increase in indent. On purpose?

Author: Thanks Rachel, your sentence is much better, so I've copied it into my next update.

The indent is not deliberate. It's something weird that my markdown editor does. I'll try to get rid of it.

1. Include a comment on the relevant repo issue that explains your intention — say what you would like to copy (include the URL) and explain why you think this is appropriate.
2. If the content is published under a closed license:

>    - If you hold the rights to the content, state this, and say that are happy to republish it on MDN

Reviewer 3: ```suggestion
   - If you hold the rights to the content, state this, and say that you are happy to republish it on MDN.
```

Author: Thanks Ryan; i've grabbed this and will use it in my next update.
